modules/doc/content/python/mms.md

```diff
@@ -0,0 +1,239 @@
+# Method of Manufactured Solutions (MMS)
+
+The method of manufactured solutions (MMS) is one method that may be used to validate that a
+system of partial differential equations (PDEs) is solving as expected. This method is general and
+should be used heavily in testing, thus some basic utilities exist within MOOSE to aid in building
+such tests.
+
+The first step in the process is to be sure that the strong form of your equation is well understood.
+For this example, a diffusion equation is being considered.
+
+\begin{equation}
+\label{strong}
+\nabla \cdot \nabla u = 0.
+\end{equation}
+
+Next, assume a solution to this equation. It is important to pick an equation that has the necessary
+order to allow for all the derivatives to be computed, both spatially and temporally.
+
+When performing a spatial convergence study for a transient problem the time portion
+should be an appropriate order that allows for the temporal finite difference scheme to represent
+solution exactly. To the contrary the spatial solution should not be exactly represented
```

Comment:
Did you find that you could do this? That you were able to eliminate possible error pollution from the temporal time integration?

Reply:
I think I did, I did this in the results from the two examples and both achieved the convergence expected.